app/assets/javascripts/search.js

```diff
@@ -0,0 +1,14 @@
+function searchIdeas(){
+  $("#search").keyup(function(event){
+    var searchParam = $(this).val().toLowerCase();
+    var allExisitingIdeas = $('#index').children();
+
+    allExisitingIdeas.removeClass('invisible');
```

Comment:
Just a thought, but would it make sense to filter the ideas first, and then only remove the invisible class from those ideas that should not be hidden?